airflow/metrics/otel_logger.py

```diff
@@ -235,6 +301,57 @@ def del_counter(self, name: str, attributes: Attributes = None) -> None:
         if key in self.map.keys():
             del self.map[key]
 
+    def set_value(self, name: str, value: float, delta: bool, tags: Attributes):
```

Comment:
minor, all the other methods are `<something>_gauge`, I feel like this one should follow suit?
```suggestion
    def set_gauge(self, name: str, value: float, delta: bool, tags: Attributes):
```

Reply:
Yeah, I guess that's fair.  I'll make the changes.